blake2.gemspec

```diff
@@ -1,26 +1,35 @@
 # coding: utf-8
 Gem::Specification.new do |spec|
-  spec.name     = "blake2"
-  spec.version  = "0.1.0"
-  spec.authors  = ["Franck Verrot"]
-  spec.email    = ["franck@verrot.fr"]
-  spec.homepage = "https://github.com/franckverrot/blake2"
-  spec.license  = "GPLv3"
+  spec.name     = 'blake2'
+  spec.version  = '0.1.0'
+  spec.authors  = ['Franck Verrot']
+  spec.email    = ['franck@verrot.fr']
+  spec.homepage = 'https://github.com/franckverrot/blake2'
+  spec.license  = 'GPLv3'
 
-  spec.summary     = "BLAKE2 - fast secure hashing - for Ruby"
+  spec.required_ruby_version = '>= 2.1.0'
+
+  spec.summary     = 'BLAKE2 - fast secure hashing - for Ruby'
   spec.description = spec.summary
-  spec.files       = `git ls-files -z`.split("\x0")
+  spec.files       = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.bindir      = 'bin'
+  # spec.executables  << ''
 
-  spec.extensions << "ext/blake2_ext/extconf.rb"
+  spec.extensions << 'ext/blake2_ext/extconf.rb'
 
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
-  spec.require_paths = ["lib"]
+  spec.require_paths = ['lib']
 
-  spec.add_development_dependency "rake-compiler"
-  spec.add_development_dependency "bundler", "~> 1.5"
-  spec.add_development_dependency "rake"
-  spec.add_development_dependency "minitest"
+  spec.add_development_dependency 'rake-compiler'
+  spec.add_development_dependency 'bundler', '~> 1.5'
+  spec.add_development_dependency 'rake'
+  spec.add_development_dependency 'minitest'
+  spec.add_development_dependency 'pry', '~> 0.10'
 
-  spec.cert_chain  = ['certs/franckverrot.pem']
-  spec.signing_key = File.expand_path(ENV['RUBYGEMS_CERT_PATH']) if $0 =~ /gem\z/
+  cert_path = ENV['RUBYGEMS_CERT_PATH']
+  cert = File.expand_path(ENV['RUBYGEMS_CERT_PATH']) if cert_path
+  if cert && File.exist?(cert)
+    spec.signing_key = cert
+    spec.cert_chain = ['certs/franckverrot.pem']
+  end
 end
```

Comment:
If any other developer, say me for example, tries to locally build the gem it will fail using the gemspec the way you have it since the cert_chain file doesn't exist on any machine but your own.

```
~/src/blake2 (fixes_20160513 ✗)$ rake build
rake aborted!
Invalid gemspec in [/Users/glenn/src/blake2/blake2.gemspec]: no implicit conversion of nil into String
ERROR:  Error loading gemspec. Aborting.

Tasks: TOP => build
(See full trace by running task with --trace)
```

The way I adjusted it should allow anyone to build the gem, but only if the files you only have exist will it prompt you for signing the gem.